projects/pgai/db/sql/idempotent/999-privileges.sql

```diff
@@ -0,0 +1,24 @@
+create or replace function ai.grant_vectorizer_usage(to_user pg_catalog.name, admin pg_catalog.bool default false) returns void
+as $func$
+begin
+    if not admin then
+        execute 'grant usage, create on schema ai to ' || to_user;
```

Comment:
The idea is that the extension will continue to have an `ai` schema, right? How does this (specifically granting `create` on `ai`) play with that?